plugins/uberdev/lib/review-push-target.sh

```
@@ -0,0 +1,100 @@
+# review-push-target.sh -- the same-repository push-target resolver for
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)